src/app/profile/overview/work-items/work-items.component.html

```diff
@@ -50,3 +105,4 @@
     </ul>
   </div>
 </ng-template>
+ -->
```

Comment:
You can also delete the commented-out code.

A return char is also needed at the end of file.